internal/views/sts.go

```diff
@@ -11,12 +11,13 @@ import (
 
 type statefulSetView struct {
 	*logResourceView
-	scalableResourceView *scalableResourceView
+	scalableResourceView    *scalableResourceView
+	restartableResourceView *restartableResourceView
 }
 
 func newStatefulSetView(title, gvr string, app *appView, list resource.List) resourceViewer {
 	logResourceView := newLogResourceView(title, gvr, app, list)
-	v := statefulSetView{logResourceView, newScalableResourceViewForParent(logResourceView.resourceView)}
+	v := statefulSetView{logResourceView: logResourceView, scalableResourceView: newScalableResourceViewForParent(logResourceView.resourceView), restartableResourceView: newRestartableResourceViewForParent(logResourceView.resourceView)}
```

Comment:
nit: break the declaration in multiples lines in the same way of `daemonSetView` and `deployView`